email_parser/reader.py

```diff
@@ -156,3 +159,13 @@ def read(email, settings):
     ordered_placeholders = _ordered_placeholders(template.placeholders_order, placeholders)
 
     return template, ordered_placeholders, ignored_plceholder_names
+
+
+def read_link_locale_mappings(settings):
+    try:
+        content = fs.read_file(settings.source, LINK_LOCALE_MAPPINGS_FILENAME)
```

Comment:
is the path to the locale mappings provided via kwargs? if so why do we assume static filename for it? better not mix convention. Either setting comes fully defined by user either it's defined static